### PR TITLE
feat: hardcode pkg version number

### PR DESCRIPTION
**Pull request by cpaniaguam**
This approach is working well with IFTPipeline!

### CODE REVIEW

**Review by tdivoll — COMMENTED** [review]
I was testing with `Pkg.project().version` instead of `VersionNumber(0,2,0)`. It seems to return the version fine and updating the test worked as well. However, it tries to activate a Julia REPL and IFT when I try to commit the changes.

File: src/IceFloeTracker.jl [hunk 1]
```diff
@@ -67,14 +67,9 @@ include("special_strels.jl")
 
 const sk_measure = PyNULL()
 const getlatlon = PyNULL()
-const _version = []
+const IFTVERSION = VersionNumber(0,2,0)
```
Comment (tdivoll) [inline src/IceFloeTracker.jl:70]:
```suggestion
const IFTVERSION = Pkg.project().version
```

File: test/test-misc.jl [hunk 2]
```diff
@@ -1,3 +1,3 @@
 println("------------------------------------------------
 ----------------- Misc. Tests ------------------")
-@test getiftversion() == "0.2.0"
+@test IFTVERSION == VersionNumber(0,2,0)
```
Comment (tdivoll) [inline test/test-misc.jl:3]:
```suggestion
@test IFTVERSION == Pkg.project().version
```

**Comment by cpaniaguam** [comment]
> I was testing with `Pkg.project().version` instead of `VersionNumber(0,2,0)`. It seems to return the version fine and updating the test worked as well. However, it tries to activate a Julia REPL and IFT when I try to commit the changes.

I'll test this within IFTPipeline. Thanks!

**Comment by cpaniaguam** [comment]
> > I was testing with `Pkg.project().version` instead of `VersionNumber(0,2,0)`. It seems to return the version fine and updating the test worked as well. However, it tries to activate a Julia REPL and IFT when I try to commit the changes.
> 
> I'll test this within IFTPipeline. Thanks!

When used within IFTPipeline it returns IFTPipeline's version :/. Going back to testing with hard-coding the version number.

**Comment by cpaniaguam** [comment]
@tdivoll I think hard coding the version number is probably the way to go.

**Comment by cpaniaguam** [comment]
> @tdivoll I think hard coding the version number is probably the way to go.

@tdivoll Things working well within the container.

**Review by tdivoll — APPROVED** [review]
Sounds good to me, it is a minor task to update when we update version numbers.

**Pull request merged**